editor/js/Viewport.Info.js

```diff
@@ -98,6 +103,30 @@ function ViewportInfo( editor ) {
 
 	}
 
+	//
+
+	editor.signals.pathTracerUpdated.add( function ( samples ) {
+
+		samples = Math.floor( samples );
+
+		samplesText.setValue( samples );
+
+		const samplesStringKey = ( pluralRules.select( samples ) === 'one' ) ? 'viewport/info/oneSample' : 'viewport/info/samples';
```

Comment:
Reconsidering #28245.

Do we really need this distinction? Stuff like that creates an unnecessary complexity. Why not just using `Sample(s)`?

Reply:
vertex(ices) then ?

Reply:
Or we just always use the plural? I personally don't see `1` samples as an issue which should be fixed with a separate code path.

Reply:
The issue about using `(s)` for just 'Sample(s)' is that, that unit will be inconsistent with other units formatted in the same area:

```
1 object
1 vertex
0 triangles
1 sample(s)
```

The reason that we can't apply `(s)` for other units has been mentioned https://github.com/mrdoob/three.js/pull/28432?notification_referrer_id=NT_kwDOABA4arMxMDcyNTk5ODMxOToxMDYzMDE4#discussion_r1606522848 so skipped.

If the plural rules handling is disturbing, then I prefer taking them into a function:

```
function formats( quantity, unit ) {...}
```

Reply:
Then I vote for always using plural (without brackets).

Reply:
Note that the transition from 0 > 1 > 2 samples is noticeable in low-end devices, so 'always use samples' will confuse users that the quantity is incorrect for 1 samples